### DISCUSSION TITLE
CAN'T register under the same phone number

### DISCUSSION THREAD

**Discussion by benjamin-heidi** [vercel/community#7399] [Account Issue] [answered]
### Question

I have already emailed vercel 3 weeks ago, but now this is still not resolved and I highly doubted if support team would help me with this.

Right now I have my personal account and my company email, I need to register both under the same phone number and vercel is blocking me from doing so. Its been nearly three weeks and I am very disappointed to see you can't even help me with this small issue in time.

**Comment by pawlean** [comment]
Hi @benjamin-heidi,

Sorry to hear this - I've passed it on internally and hope to get an answer to you as soon as we can.

**Reply by jamesb-tacgov** [reply]
This is a problem for me as well

**Comment by amyegan** [comment, marked as answer]
Hi @benjamin-heidi and @jamesb-tacgov. We can't safely discuss account information in public because of security concerns. For account registration issues, please email [registration@vercel.com](mailto:registration@vercel.com) to create a new ticket (if you haven't already done so). Be sure to explain the situation and let them know about email addresses and phone number used for your accounts.

Alternatively, you can now include both personal and work email addresses on the same Vercel account: https://vercel.com/changelog/accounts-can-now-have-multiple-email-addresses
